app/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.neenbedankt.android-apt'
 apply plugin: 'hu.supercluster.paperwork'
 apply from: '../code_quality_tools/jacoco.gradle'
 apply plugin: 'com.frogermcs.androiddevmetrics'
+apply from: '../code_quality_tools/quality.gradle'
```

Comment:
I'm not sure, but I don't see anything that won't work for java module :)

Probably java plugin puts classes to different folder btw `classes = files("${project.projectDir}/build/intermediates/classes")`